src/server.ts

```diff
@@ -312,9 +312,14 @@ app.post('/edit/preview',
 		};
 		word["na'vi"] = word['word_raw']['FN'];
 
-		let dataErrorList: string[] = [];
+		let dataErrorList: DataIssue[] = [];
 		reykunyu.preprocessWord(word, dataErrorList);
-		if (dataErrorList.length > 0) {
+		let errorCount = 0;
+		for (let error of dataErrorList) {
+			if (error.type === 'error')
```

Comment:
Note to self (not relevant for this PR): it makes sense to only throw an error here for ... errors, but we should eventually also show warnings in the editor (while still allowing to save the word).

I also have like 10 more ideas for things we could warn for hrh